executor/distsql.go

```diff
@@ -437,22 +451,17 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 		return errors.Trace(err)
 	}
 	// Since the first read only need handle information. So its returned col is only 1.
-	result, err := distsql.SelectWithRuntimeStats(ctx, e.ctx, kvReq, []*types.FieldType{types.NewFieldType(mysql.TypeLonglong)}, e.feedback, getPhysicalPlanIDs(e.idxPlans))
+	result, err := e.SelectResult(ctx, e.ctx, kvReq, []*types.FieldType{types.NewFieldType(mysql.TypeLonglong)}, e.feedback, getPhysicalPlanIDs(e.idxPlans))
 	if err != nil {
 		return errors.Trace(err)
 	}
 	result.Fetch(ctx)
 	worker := &indexWorker{
-		workCh:       workCh,
-		finished:     e.finished,
-		resultCh:     e.resultCh,
-		keepOrder:    e.keepOrder,
-		batchSize:    e.maxChunkSize,
-		maxBatchSize: e.ctx.GetSessionVars().IndexLookupSize,
```

Comment:
we can also remove indexLookupSize from session and configuration? and also update the document to invalidate this configuration item.

it's almost good to me, but remove batchSize maybe will make performance degrade when there are big region num + small result set + without limit?

Reply:
According to the advice from @zz-jason, we should keep `IndexLookupSize` and take it into account while setting `RequiredRows`.
So how about set the RequiredRows in indexWorker to min(tidb_index_lookup_size, RequiredRows from parent)?